include/openssl/tserr.h

```diff
@@ -92,9 +86,9 @@ int ERR_load_TS_strings(void);
 #  define TS_R_COULD_NOT_SET_ENGINE                        127
 #  define TS_R_COULD_NOT_SET_TIME                          115
 #  define TS_R_DETACHED_CONTENT                            134
-#  define TS_R_ESS_ADD_SIGNING_CERT_ERROR                  116
```

Comment:
Nit: The sum of previous of changes and revert has created an useless modification of this 3 lines.
Restore previous values, and run a `make update,` it will revert a few others changes related to this one.